### PR TITLE
Warm up events.

### PR DESCRIPTION
**Pull request by cramforce**

### CODE REVIEW

File: contents/news/schedule.md [hunk 1]
```diff
@@ -12,8 +12,12 @@ Below you'll find a high level overview of the events. See the **[full timetable
 ## Before the conference
 
 - Week before and after the conference: **[WhatTheFest Berlin](http://wwwtf.berlin/)**.
+- May 31st, Thursday, 19:00: CSSconf EU welcome evening (open to JSConf EU attendees). [Info](/news/warm-up-events/#thursday-may-31st-19-00)
```
Comment (Kriesse) [inline contents/news/schedule.md]: FYI, official name (sponsor pays for it) is "CSSconf Warmup sponsored by talent.io"
But fine if on the JSConf site you opt for something else.

File: contents/news/schedule.md [hunk 2]
```diff
@@ -12,8 +12,12 @@ Below you'll find a high level overview of the events. See the **[full timetable
...
+
+## June 1, Friday
+
 - June 1, Friday: **[CSSConf EU (Tickets sold separately)](https://2018.cssconf.eu/)**.
-- June 1, Friday: **JSConf EU Opening Party**, Location TBA, from 20:00.
+- June 1, Friday at 20:00: **JSConf EU Opening Party and Pre-Registration**. [Info](/news/warm-up-events/#friday-june-1st-jsconf-eu-pre-party-20-00).
```
Comment (Kriesse) [inline contents/news/schedule.md]: 21:00 (not 20:00 - to give artribute enough time to wrap up catering and dinner)

File: contents/news/warm-up-events.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+title: Warm Up Events
+date: 2018-05-22
+image: https://payload204.cargocollective.com/1/13/433503/6394823/OberhafenkantineReplikaBerlin_1340_c.jpg
+----
+
+In case you're arriving a few days early, we'd love to meet you at these events:
+
+## Thursday, May 31st, 19:00
+
+CSSconf EU welcome evening (all JSConf attendees welcome) at [Oberhafenkantine](http://www.oberhafenkantine-berlin.de/).
```
Comment (Kriesse) [inline contents/news/warm-up-events.md]: "CSSconf Warmup sponsored by talent.io", see above

File: contents/news/warm-up-events.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+![Image of Oberhafenkantine Berlin](https://payload204.cargocollective.com/1/13/433503/6394823/OberhafenkantineReplikaBerlin_1340_c.jpg)
+
+[Oberhafenkantine Berlin, Am Flutgraben 2, 12435 Berlin, Germany](https://goo.gl/maps/pesFTY3qXRE2)
+
+## Friday June 1st, CSSconf EU
+
+If you're in Berlin on Friday, [consider going to CSSconf EU.](https://2018.cssconf.eu/)
+
+## Friday June 1st, JSConf EU Pre Party, 20:00
```
Comment (Kriesse) [inline contents/news/warm-up-events.md]: 21:00 (not 20:00 - to give artribute enough time to wrap up catering and dinner)

File: contents/news/schedule.md (hunk 1 above)
Comment (cramforce) [reply contents/news/schedule.md]: Done

File: contents/news/schedule.md (hunk 2 above)
Comment (cramforce) [reply contents/news/schedule.md]: Done

File: contents/news/warm-up-events.md (hunk 3 above)
Comment (cramforce) [reply contents/news/warm-up-events.md]: Done

File: contents/news/warm-up-events.md (hunk 4 above)
Comment (cramforce) [reply contents/news/warm-up-events.md]: Done

**Pull request merged**